t/t1018-mf-priority-disable-entry.t

```diff
@@ -0,0 +1,114 @@
+	flux account-priority-update -p $(pwd)/FluxAccountingTest.db
+'
+
+test_expect_success 'submit a job successfully under default bank' '
+	jobid1=$(flux mini submit -n1 hostname) &&
+	flux job wait-event -f json $jobid1 priority | jq '.context.priority' > job1.test &&
+	cat <<-EOF >job1.expected &&
+	10050000
+	EOF
+	test_cmp job1.expected job1.test
+'
+
+test_expect_success 'submit a job successfully under second bank' '
+	jobid2=$(flux mini submit --setattr=system.bank=account2 -n1 hostname) &&
+	flux job wait-event -f json $jobid2 priority | jq '.context.priority' > job2.test &&
+	cat <<-EOF >job2.expected &&
+	10050000
+	EOF
+	test_cmp job2.expected job2.test
+'
+
+test_expect_success 'disable second user/bank entry and update plugin' '
+	flux account -p ${DB_PATH} delete-user $username account2 &&
+	flux account-priority-update -p $(pwd)/FluxAccountingTest.db
+'
+
+test_expect_success 'try to submit job under second user/bank entry' '
+	test_must_fail flux mini submit --setattr=system.bank=account2 -n1 hostname > deleted_entry1.out 2>&1 &&
+	test_debug "cat deleted_entry1.out" &&
+	grep "user/bank entry has been disabled from flux-accounting DB" deleted_entry1.out
+'
+
+test_expect_success 're-add second user/bank entry and update-plugin' '
+	flux account -p ${DB_PATH} add-user --username=$username --userid=$uid --bank=account2 --shares=1 &&
+	flux account-priority-update -p $(pwd)/FluxAccountingTest.db
+'
+
+test_expect_success 'submit a job successfully under second bank' '
+	jobid3=$(flux mini submit --setattr=system.bank=account2 -n1 hostname) &&
+	flux job wait-event -f json $jobid3 priority | jq '.context.priority' > job3.test &&
+	cat <<-EOF >job3.expected &&
+	10050000
+	EOF
+	test_cmp job3.expected job3.test
+'
+
+test_expect_success 'disable first (and default) bank entry for user (will update the plugin with a new default bank)' '
+	flux account -p ${DB_PATH} delete-user $username account1 &&
+	flux account-priority-update -p $(pwd)/FluxAccountingTest.db
+'
+
+test_expect_success 'try to submit job under new default user/bank entry' '
+	jobid4=$(flux mini submit -n1 hostname) &&
+	flux job wait-event -f json $jobid4 priority | jq '.context.priority' > job4.test &&
+	cat <<-EOF >job4.expected &&
+	10050000
+	EOF
+	test_cmp job4.expected job4.test
+'
+
+test_expect_success 'disabling a user while they have an active job should not kill the job' '
+	jobid5=$(flux mini submit -n1 sleep 60) &&
+	flux account -p ${DB_PATH} delete-user $username account2 &&
+	flux account-priority-update -p $(pwd)/FluxAccountingTest.db &&
+	test $(flux jobs -no {state} ${jobid5}) = RUN &&
+	flux job cancel $jobid5
+'
+
+test_expect_success 'trying to submit a job now should result in a job rejection' '
+	test_must_fail flux mini submit --setattr=system.bank=account2 -n1 hostname > deleted_entry2.out 2>&1 &&
+	test_debug "cat deleted_entry2.out" &&
+	grep "user/bank entry has been disabled from flux-accounting DB" deleted_entry2.out
+'
+
```

Comment:
Maybe add a test that re-activates the user/bank and ensures jobs can be submitted again?

Reply:
I believe I have a couple of tests that go through this scenario earlier in the test file:

```
test_expect_success 're-add second user/bank entry and update-plugin' '
	flux account -p ${DB_PATH} add-user --username=$username --userid=$uid --bank=account2 --shares=1 &&
	flux account-priority-update -p $(pwd)/FluxAccountingTest.db
'

test_expect_success 'submit a job successfully under second bank' '
	jobid3=$(flux mini submit --setattr=system.bank=account2 -n1 hostname) &&
	flux job wait-event -f json $jobid3 priority | jq '.context.priority' > job3.test &&
	cat <<-EOF >job3.expected &&
	10050000
	EOF
	test_cmp job3.expected job3.test
'
```

Sorry if you meant something else though (or in addition to what I had before 😅)!

Reply:
Ah, sorry I missed that! LGTM then!